### ISSUE TITLE
[Bootstrap] Step 11: Deploy ops-depls jobs manually with Concourse -  bosh-ops failed due to the openstack CPI missing

### ISSUE THREAD

**Issue by wdesplas** [orange-cloudfoundry/paas-templates#160]
**Observed behavior**

The bosh-ops deployment failed. In fact, the openstack cpi is missing:
```
Launch bosh-ops job but got :
Task 240 | 14:41:57 | Preparing deployment: Preparing deployment (00:00:00)
                    L Error: Release 'bosh-openstack-cpi' doesn't exist
Task 240 | 14:41:57 | Error: Release 'bosh-openstack-cpi' doesn't exist
```

**Workaround**
Uploading the openstack CPI manually:

```
bosh upload-release  https://bosh.io/d/github.com/cloudfoundry/bosh-openstack-cpi-release?v=37
```

**Comment by o-orand** [comment]
The corresponding issue in orange-cloudfoundry/cf-ops-automation#204

**Comment by o-orand** [comment]
Included in COA 3.2.0
